Library/Formula/sbuild.rb

```diff
@@ -1,13 +1,11 @@
-require 'formula'
-
 class Sbuild < Formula
-  homepage 'http://sbuild.org'
-  url 'http://sbuild.org/uploads/sbuild/0.7.6/sbuild-0.7.6-dist.zip'
-  sha1 'd3c815eea2db0347c2b8a319d52c32a50b03d803'
+  homepage "http://sbuild.org"
+  url "http://sbuild.org/uploads/sbuild/0.7.6/sbuild-0.7.6-dist.zip"
+  sha256 "64e17df4db26980170e5cf616d015924da01f232c5207f2bdc7ba10e7d976e3d"
 
   def install
-    libexec.install Dir['*']
-    system "chmod +x #{libexec}/bin/sbuild"
+    libexec.install Dir["*"]
+    chmod 0755, "#{libexec}/bin/sbuild"
```

Comment:
You should be able to change this to ` chmod 0755, libexec/"bin/sbuild" `. It's just a style thing really, though.